Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/ui/details/RecoveryCertificateDetailsFragment.kt

```diff
@@ -59,7 +66,7 @@ class RecoveryCertificateDetailsFragment : Fragment(R.layout.fragment_recovery_c
 
         fullname.text = certificate.fullName
         dateOfBirth.text = certificate.dateOfBirthFormatted
-        recoveredFromDisease.text = certificate.targetDisease
+        recoveredFromDesease.text = certificate.targetDisease
```

Comment:
Previous spelling was correct?

Reply:
@axelherbstreith  I think you renamed it ?

Reply:
@d4rken Yes _Disease_ is the right spelling, there was one attribute with was wrong e.g. _Desease_, which I wanted to fix, maybe I messed something up there, will be changed 👍